sdk/core/core/inc/az_span.h

```diff
@@ -118,13 +129,21 @@ AZ_NODISCARD az_span az_span_from_str(char* str);
     }, \
   }
 
+/**
+ * @brief AZ_SPAN_FROM_BUFFER returns an az_span expression over an unitialized byte buffer. For
+ * example:
+ *
+ * uint8_t buffer[1024];
+ * some_function(AZ_SPAN_FROM_BUFFER(buffer));  // Len=0, Cap=1024
+ */
 #define AZ_SPAN_FROM_BUFFER(BYTE_BUFFER) (az_span) AZ_SPAN_LITERAL_FROM_BUFFER(BYTE_BUFFER)
 
 /**
- * Creates a az_span literal which can be used to initialize a constant. For example
+ * @brief AZ_SPAN_LITERAL_FROM_INITIALIZED_BUFFER returns a literal az_span over an initialized byte
+ * buffer. For example:
  *
- * uint8_t buffer[] = {"Hello World"};
- * const az_span foo = AZ_SPAN_LITERAL_FROM_BUFFER(buffer);
+ * uint8_t buffer[] = { 1, 2, 3 };
+ * const az_span buf = AZ_SPAN_LITERAL_FROM_INITIALIZED_BUFFER(buffer); // Len=3, Cap=3
  */
 #define AZ_SPAN_LITERAL_FROM_INITIALIZED_BUFFER(BYTE_BUFFER) \
```

Comment:
Can we remove this and only keep `AZ_SPAN_FROM_INITIALIZED_BUFFER`?

Reply:
No